### ISSUE TITLE
AMMEX: keep focus on planned transactions

### ISSUE THREAD

**Issue by jeepcook** [moneymanagerex/android-money-manager-ex#1706]
When we are in planned transactions menu, select a transaction and do a back, the "focus" is not registered so we have on the screen the first transaction and others following the first one. not the same screen as previous one.

The fix should be to implement the same functionality as the one used for accounts which go back to the same previous screen as the one seen before opening a transaction.

AMMEX 20240509-1032

**Comment by jeepcook** [comment]
Hello,
What about a fix for this issue. Why is 1743 is linked to it? It's not a notification problem.
Thanks

**Comment by wolfsolver** [comment]
becouse in my test it was fixed during recurring notification implementation. But make a deep test now, the issue is still present. The point is that since date in edit transaction can ben changed, the position of edited recurring transaction can be different than previus.

**Comment by jeepcook** [comment]
Hello,
Is there any hope of fixing this bug?
Thanks

**Comment by jeepcook** [comment]
The issue is still there and this one has to stay open.

**Comment by jeepcook** [comment]
The issue is still there. Other screens have been fixed, not this one, I don't know why. So issue must remain open.

**Comment by jeepcook** [comment]
Comment to stay issue open.

**Comment by jeepcook** [comment]
Hello, 
Near two years this issue is opened. Nobody want to work on a solution?
If it's the case it's understandable, but I will close definitely the issue.
Thanks

**Comment by JoshuaLampert** [comment]
I don't understand what exactly the issue is. When I select a transaction in the scheduled transaction list I see a list of different options. When I do a back, I am back at the list of scheduled transaction (the same screen as before), i.e., what I would expect. Also when I enter the next occurrence and do a back, everything is as expected for me. Can you describe the exact steps to reproduce this?

**Comment by jeepcook** [comment]
It's just because you don't have enough transactions to reproduce the problem. I have next to 50 scheduled transactions. If I scroll to a transaction not in the first screen but in the last screen for example and go back, the screen returns in the first screen not in the screen where the last selected  transaction is.
It's a problem we had long time ago in the bank accounts, the focus didn't work and we had the same operation as described above. But this has been fixed.

**Comment by JoshuaLampert** [comment]
Ah, you mean that when going back it automatically scrolls up again to the top?

**Comment by jeepcook** [comment]
Yes it's the problem

**Comment by JoshuaLampert** [comment]
Ok, I see. Yes, that is a small problem. Let me see if I (well, AI...) can fix it.

**Comment by JoshuaLampert** [comment]
I tested that #2938 fixes this issue.

**Comment by jeepcook** [comment]
Great, thanks for the fix. Will it be included in the next final release?

**Comment by JoshuaLampert** [comment]
Yes, I think so. When do you plan to release the next version, @wolfsolver?

**Comment by wolfsolver** [comment]
I can build test on June 2sd

**Comment by jeepcook** [comment]
Tested the new 5.5.8 version. Now everything is ok, thanks !
